docs/user_guide/common/Update_and_Deploy_Local_Environment.inc

```diff
@@ -1,53 +1,50 @@
 .. _ug-update-and-deploy-using-local-repos:
 
-#. Generate the new ``Puppetfile.simp``, a Puppetfile containing the latest
-   versions of only SIMP-packaged Puppet modules:
+  * Generate the Environment's Puppetfile.simp
 
-   .. code-block:: sh
+    Run :program:`simp` to pull all the latest versions of the SIMP-packaged Puppet modules
```

Comment:
Some places in this PR you use 'Puppet modules' and in others you use 'puppet modules'.  Don't know which is preferred, but would like to see some consistency.